dspace-server-webapp/src/main/java/org/dspace/app/rest/authorization/Authorization.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.authorization;
+
+import org.dspace.app.rest.model.BaseObjectRest;
+import org.dspace.content.Site;
+import org.dspace.discovery.IndexableObject;
+import org.dspace.eperson.EPerson;
+
+/**
+ * An authorization is the right for a specific {@link EPerson}, eventually null to indicate unauthenticated users, to
+ * use a specific {@link AuthorizationFeature} on a defined object. The target object must implement the
+ * {@link IndexableObject} interface so to have an unique ID and type
```

Comment:
The IndexableObject is mentioned here, but it isn't used in this class, I assume it should be the "BaseObjectRest" ?

Reply:
fixed thanks